.github/workflows/test-application.yaml

```diff
@@ -69,6 +66,9 @@ jobs:
           --health-cmd "pg_isready" --health-interval 10s --health-timeout 5s --health-retries 5
 
     steps:
+      - name: start
+        run: docker run -d mysql:8.3 --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci
```

Comment:
if i understand correctly, this starts another mysql instance but we keep using the one from services.